src/tasks/tasks.py

```diff
@@ -2263,23 +2263,36 @@ def load_data(self):
 
         trg_map = {"true": 1, "false": 0, True: 1, False: 0}
 
+        def _process_sent_special(sent, word):
```

Comment:
More descriptive name? Why 'special'?

Reply:
We want to track the indices of the marked word after processing so that we can extract its representation.